tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -7,11 +7,17 @@
 
 
 class StatusBar(Base):
+    _status_bar_locator = (By.ID, 'statusbar')
```

Comment:
If we move the status bar to a page Region, this ID would have its place in the System App.